Documentation/networking/sch_skbprio.txt

```diff
@@ -0,0 +1,24 @@
+SKB Priority Queue
+=================================
```

Comment:
Feel free to ignore this suggestion since it's just aesthetics: make the length of the line of equals the same as the title:
```
SKB Priority Queue
==================
```